### PR TITLE
Fix image loading in tutorials code

### PR DESCRIPTION
**Pull request by LaurentBerger**
Fixes https://github.com/opencv/opencv/issues/23378

### Pull Request Readiness Checklist

See details at https://github.com/opencv/opencv/wiki/How_to_contribute#making-a-good-pull-request

- [X] I agree to contribute to the project under Apache 2 License.
- [X] To the best of my knowledge, the proposed patch is not based on a code under GPL or another license that is incompatible with OpenCV
- [X] The PR is proposed to the proper branch
- [X] There is a reference to the original bug report and related work
- [X] There is accuracy test, performance test and test data in opencv_extra repository, if applicable
      Patch to opencv_extra has the same branch name.
- [X] The feature is well documented and sample code can be built with the project CMake

### CODE REVIEW

File: samples/cpp/tutorial_code/ImgProc/out_of_focus_deblur_filter/out_of_focus_deblur_filter.cpp [hunk 1]
```diff
@@ -17,9 +18,9 @@ void calcWnrFilter(const Mat& input_h_PSF, Mat& output_G, double nsr);
 
 const String keys =
 "{help h usage ? |             | print this message   }"
-"{image          |original.JPG | input image name     }"
-"{R              |53           | radius               }"
-"{SNR            |5200         | signal to noise ratio}"
+"{image          |original.jpg | input image name     }"
+"{R              |5           | radius               }"
+"{SNR            |100         | signal to noise ratio}"
```
[the remark this answers is not in the document]
Comment (LaurentBerger) [reply samples/cpp/tutorial_code/ImgProc/out_of_focus_deblur_filter/out_of_focus_deblur_filter.cpp:23]: because previous values does not work

**Comment by asmorkalov** [comment]
Tried manually with Ubuntu 20.04.Works well for me. Thanks a lot for the contribution!

**Comment by LaurentBerger** [comment]
> Tried manually with Ubuntu 20.04.Works well for me. Thanks a lot for the contribution!

On windows 
example_tutorial_out_of_focus_deblur_filter.exe -R=52 -SNR=5200
image

and with example_tutorial_out_of_focus_deblur_filter.exe -R=5 -SNR=100

image

**Review by asmorkalov — APPROVED** [review]
:+1:

**Pull request merged**